src/Form/index.tsx

```diff
@@ -878,71 +880,74 @@ function Form({
   const [getCardElement, setCardElement] = usePayments();
 
   async function setupStepPaymentMethod(formattedFields: any) {
-    for (let i = 0; i < activeStep.servar_fields.length; i++) {
-      const servar = activeStep.servar_fields[i].servar;
-      if (servar.type === 'payment_method') {
-        const integrationData = integrations?.stripe;
-        const actionData: ActionData = {
-          servar,
-          client,
-          formattedFields,
-          updateFieldValues,
-          integrationData,
-          targetElement: getCardElement(servar.key)
-        };
-        const actionResult = await setupPaymentMethod(actionData);
-        return actionResult ?? {};
+    if (!_draft)
+      for (let i = 0; i < activeStep.servar_fields.length; i++) {
+        const servar = activeStep.servar_fields[i].servar;
+        if (servar.type === 'payment_method') {
+          const integrationData = integrations?.stripe;
+          const actionData: ActionData = {
+            servar,
+            client,
+            formattedFields,
+            updateFieldValues,
+            integrationData,
+            targetElement: getCardElement(servar.key)
+          };
+          const actionResult = await setupPaymentMethod(actionData);
+          return actionResult ?? {};
+        }
       }
-    }
     return {};
   }
 
   async function collectPaymentAction(triggerElement: any) {
-    const errorCallback = getErrorCallback({
-      // could be container or button but model as button for the time being...
-      trigger: lookUpTrigger(activeStep, triggerElement.id, 'container')
-    });
-    // validate all step fields and buttons.  Must be valid before payment.
-    const { invalid, inlineErrors: newInlineErrors } = validateElements({
-      step: activeStep,
-      visiblePositions,
-      triggerErrors: true,
-      errorType: formSettings.errorType,
-      formRef,
-      errorCallback,
-      setInlineErrors
-    });
-    if (invalid) return false;
-
-    // payment/checkout
-    const pm = activeStep.servar_fields.find(
-      ({ servar: { type } }: any) => type === 'payment_method'
-    );
-    const errors = await collectPayment({
-      triggerElement,
-      triggerElementType: isElementAButtonOnStep(activeStep, triggerElement)
-        ? 'button'
-        : 'container',
-      servar: pm ? pm.servar : null,
-      client,
-      updateFieldValues,
-      integrationData: integrations?.stripe,
-      targetElement: pm ? getCardElement(pm.servar.key) : null
-    });
-    if (errors) {
-      const { errorMessage, errorField } = errors;
-      await setFormElementError({
+    if (!_draft) {
```

Comment:
why does this draft check need to be here? even if we can't trigger integrations, can't we fetch the integrations for the draft form?

Reply:
I don't understand the question.  I think we are in agreement that we don't want to trigger integrations.  For Stripe this means making the card setup and the collect payment inert.  That is what I am doing with this check.  What do you mean by "fetch the integrations"?